plugins/catalog-import/src/components/EntityListComponent/EntityListComponent.tsx

```diff
@@ -140,7 +150,11 @@ export const EntityListComponent = (props: EntityListComponentProps) => {
                       : {})}
                   >
                     <ListItemIcon>{Icon && <Icon />}</ListItemIcon>
-                    <ListItemText primary={humanizeEntityRef(entity)} />
+                    <ListItemText
+                      primary={
+                        <EntityDisplayName hideIcon entityRef={entity} />
```

Comment:
OK one final thing. The presentation includes an icon for a reason - so that users can tweak the iconography. Can you change up how this is rendered, so that it actually doesn't have the ListItemIcon above but instead makes the EntityDisplayName responsible for rendering the icon?

Reply:
So by doing that UI does not look good because second and third icon looks smaller compare to first location icon. If you want to render it that then I will do the changes.

Reply:
Yeah it at least renders those rows consistently with how all other entity references are rendered, it's probably ok

Reply:
![image](https://github.com/backstage/backstage/assets/77385983/7dfbc5be-70eb-4647-9c14-53acf4655758)